cypress/e2e/data-import/settings/attach-profiles-to-job-profile.cy.js

```diff
@@ -0,0 +1,142 @@
+import MatchProfileView from '../../../support/fragments/data_import/match_profiles/matchProfileView';
+
+describe('data-import', () => {
+  describe('Settings', () => {
+    let user;
+    const matchProfile1 = {
+      profileName: `C11139 autotest match profile ${getRandomPostfix()}`,
+    };
+    const matchProfile2 = {
+      profileName: `C11139 autotest match profile ${getRandomPostfix()}`,
+    };
+    const actionProfile1 = {
+      name: `C11139 autotest action profile1 ${getRandomPostfix()}`,
+    };
+    const actionProfile2 = {
+      name: `C11139 autotest action profile2 ${getRandomPostfix()}`,
+    };
+    const actionProfile3 = {
+      name: `C11139 autotest action profile3 ${getRandomPostfix()}`,
+    };
+    const mappingProfile = {
+      name: `C11116 mapping profile ${getRandomPostfix()}`,
+    };
+    const jobProfile = {
+      profileName: `C11139 autotest job profile ${getRandomPostfix()}`,
+      acceptedType: ACCEPTED_DATA_TYPE_NAMES.MARC,
+    };
+    before('Create test data', () => {
+      cy.createTempUser([Permissions.settingsDataImportEnabled.gui]).then((userProperties) => {
+        user = userProperties;
+        cy.login(user.username, user.password, {
+          path: SettingsMenu.jobProfilePath,
+          waiter: JobProfiles.waitLoadingList,
+        });
+
+        // create 3 action profiles linked to mapping profile
+        NewFieldMappingProfile.createMappingProfileViaApi(mappingProfile.name).then(
+          (mappingProfileResponse) => {
+            NewActionProfile.createActionProfileViaApi(
+              actionProfile1.name,
+              mappingProfileResponse.body.id,
+            ).then((actionProfileResponse) => {
+              actionProfile1.id = actionProfileResponse.body.id;
+            });
+            NewActionProfile.createActionProfileViaApi(
+              actionProfile2.name,
+              mappingProfileResponse.body.id,
+            ).then((actionProfileResponse) => {
+              actionProfile2.id = actionProfileResponse.body.id;
+            });
+            NewActionProfile.createActionProfileViaApi(
+              actionProfile3.name,
+              mappingProfileResponse.body.id,
+            ).then((actionProfileResponse) => {
+              actionProfile3.id = actionProfileResponse.body.id;
+            });
+          },
+        );
+
+        // create 2 match profile
+        NewMatchProfile.createMatchProfileViaApi(matchProfile1.profileName).then(
+          (matchProfileResponse) => {
+            matchProfile1.id = matchProfileResponse.body.id;
+          },
+        );
+        NewMatchProfile.createMatchProfileViaApi(matchProfile2.profileName).then(
+          (matchProfileResponse) => {
+            matchProfile2.id = matchProfileResponse.body.id;
+          },
+        );
+      });
+    });
+
+    after('Delete test data', () => {
+      JobProfiles.deleteJobProfile(jobProfile.profileName);
+      ActionProfiles.deleteActionProfile(actionProfile1.name);
```

Comment:
And you can use 
collectionProfiles.forEach((profile) => {
        ActionProfiles.deleteActionProfile(profile.actionProfile.name);
        FieldMappingProfileView.deleteViaApi(profile.mappingProfile.name);
      });